tdrs-backend/wait_for_services.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+wait-for-it \
```

Comment:
Will ensure both of these services are ready before allowing docker startup for the `web` container.